js/web/components/SearchDropdown.js

```diff
@@ -86,11 +101,17 @@ const SearchDropdown = ({
               onKeyDown={onKeyDown}
               key={index}
             >
-              <a key={index} id={row.category} onClick={clickHandler}>
-                <Typography id={row.category} data-value={row?.name}>
-                  {row?.displayName}
-                </Typography>
-              </a>
+              <Link href={row?.link} passHref>
```

Comment:
this looks a bit odd to me having an `onClick` and the `href` attribute -- was going to say should just be onlick but am not certain what best practices are regarding ADA. we should look into that and chose a pattern